pkg/runtime/runtime.go

```diff
@@ -1382,6 +1417,15 @@ func (a *DaprRuntime) extractComponentCategory(component components_v1alpha1.Com
 	return ""
 }
 
+func (a *DaprRuntime) componentSupportRetries(componentCategory ComponentCategory) bool {
```

Comment:
This is why we should use Metadata and not add implementation specific attributes in the CRD.

Reply:
@yaron2 It is odd to have this spec only for pubsub. We don't have spec per component type, so it will pollute the spec with aggregation of all per building block settings. Unless we want retry to be valid for all building blocks.